src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -48,6 +48,48 @@
  * &lt;module name=&quot;DefaultComesLast&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
```

Comment:
Why are these changes here? They are not related to ImportOrder check.

Reply:
@rdiachenko I will revert the changes, I accidentally pushed changes for different issue to this PR